lib/Backend/GlobOpt.cpp

```diff
@@ -527,6 +532,9 @@ GlobOpt::ForwardPass()
     // Make sure we free most of them.
     Assert(freedCount >= spilledCount);
 
+    JitAdelete(alloc, this->changedSymsAfterIncBailoutCandidate);
```

Comment:
That Arena goes away right after returning from this function.  There is no need to free this.